packages/cli/test/unit/PermissionChecker.test.ts

```diff
@@ -364,6 +364,36 @@ describe('PermissionChecker.checkSubworkflowExecutePolicy', () => {
 		});
 		expect(PermissionChecker.checkSubworkflowExecutePolicy(subworkflow, userId, userId)).resolves;
 	});
+
+	test('should not throw when workflow policy is set to any', async () => {
+		const userId = 'abcde';
+		const fakeUser = { id: userId };
+		jest
+			.spyOn(UserManagementHelper, 'getWorkflowOwner')
+			.mockImplementation(async (workflowId) => fakeUser);
+		jest.spyOn(UserManagementHelper, 'isSharingEnabled').mockReturnValue(true);
+		jest.spyOn(UserService, 'get').mockImplementation(async () => fakeUser);
+		jest.spyOn(WorkflowsService, 'getSharing').mockImplementation(async () => {
+			return {
+				role: {
+					name: 'not owner',
+				},
+			};
+		});
```

Comment:
Not sure if intentional, but the three `mockImplementation` calls are throwing type errors because the mocks do not satisfy the types of the original implementation.